eds/scripts/utils.js

```diff
@@ -234,8 +244,19 @@ function getComplexQueryParams(el) {
 export function getPartnerDataCookieObject(programType) {
```

Comment:
we need to rename this. The current name makes no sense. also, the code seems to be basically the same you have above, right?